var/spack/repos/builtin/packages/grpp/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Grpp(CMakePackage):
+    """GRPP: A library for the evaluation of molecular integrals
+    of the generalized relativistic pseudopotential operator
+    (GRPP) over Gaussian functions."""
+
+    #
+    # The package has no official version and did not see a lot
+    # of development activities in 2024. It is used in cp2k and
+    # is needed when cp2k is build with trexio support
+    #
+
+    homepage = "https://github.com/aoleynichenko/libgrpp"
+    git = "https://github.com/aoleynichenko/libgrpp.git"
+
+    maintainers("mtaillefumier")
+
+    license("BSD-3-Clause")
+
+    build_system("cmake")
+
+    version("main", branch="main")
+    version(
+        "2023.12.25", commit="64d157f1dc95815096b1fd437a5851abeb3425929cf7b2092bf8262db9c5e33d"
```

Comment:
Just checking, this version number is "made up" (matches the date of the commit), correct? The date format was chosen because it matches what is used in CP2K?

Reply:
I missed it at the first pass, but does this commit actually exist? Looks like the SHA of the package used in CP2K.

The version is indeed what CP2K calls it, but I'm not sure which commit it corresponds to... https://github.com/cp2k/cp2k/blob/26096efc000316683be093141185a0bd14d2043f/tools/toolchain/scripts/stage3/install_libgrpp.sh#L9-L11

Reply:
Good point. It looks this is the sha256 sum of some tarball that cp2k knows how to download, not a commit hash?

Reply:
Updating review to block merging until this is resolved.

Reply:
I changed the hash to the specific commit after @RMeli contacted me privately.